gatling-jms/src/main/scala/io/gatling/jms/check/JmsMessageBodyPreparers.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2011-2019 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gatling.jms.check
+
+import io.gatling.core.check.Preparer
+import javax.jms.{ BytesMessage, Message, TextMessage }
+import io.gatling.commons.validation._
+
+object JmsMessageBodyPreparers {
+
+  val JmsStringBodyPreparer: Preparer[Message, String] = {
+    case tm: TextMessage  => tm.getText.success
+    case bm: BytesMessage => bm.readUTF.success
+    case _                => "Unsupported message type".failure
+  }
+
+  val JmsBytesBodyPreparer: Preparer[Message, Array[Byte]] = {
+    case tm: TextMessage => tm.getText.getBytes.success
```

Comment:
String#getBytes uses JVM's default encoding. We can't assume it's the correct one, we must use the one in GatlingConfiguration.